app/experimenter/legacy-ui/rapid/components/experiments/ExperimentDetails.tsx

```diff
@@ -279,6 +280,13 @@ const ExperimentDetails: React.FC = () => {
           </tbody>
         </table>
 
+        {experimentData.analysis?.show_analysis && (
+          <>
+            <h2>Overview</h2>
+            <HighlightsTable experimentData={experimentData} />
```

Comment:
I know @LZoog wouldn't let me live it down if I didn't mention this: you can spread same name properties! `<HighlightsTable {...{ experimentData }} />`

Reply:
Crazy

Reply:
wow neat!